pkg/kubelet/cri/remote/remote_runtime.go

```diff
@@ -128,7 +128,7 @@ func (r *remoteRuntimeService) useV1API() bool {
 // being upgraded, then the container runtime must also support the initially
 // selected version or the redial is expected to fail, which requires a restart
 // of kubelet.
-func (r *remoteRuntimeService) determineAPIVersion(conn *grpc.ClientConn) error {
+func (r *remoteRuntimeService) determineAPIVersion(conn *grpc.ClientConn, endpoint string) error {
```

Comment:
I didn't see the endpoint is used anywhere besides for logging purpose. Shall we keep it as before ?

Reply:
I see it's in newest Kubernetes [version](https://github.com/kubernetes/kubernetes/blob/master/pkg/kubelet/cri/remote/remote_runtime.go#L131), can remove however.

Reply:
I see. In this case, you can sync your branch, then it won't appear in the diff